inst/extdata/aglu/Various_ag_resbio_data.csv

```diff
@@ -1,7 +1,8 @@
 # File: Various_ag_resbio_data.csv,,,,,
+# Title: Various_ag_resbio_data,,,,,
+# Units: Varied,,,,,

```

Comment:
Not all units are specified in the input file or the original code. The five parameters are: HarvestIndex, ErosCtrl_tHa, ResEnergy_GJt, Root_Shoot, and WaterContent. So I guess ErosCtrl is in ton/Ha and ResEnergy is in GJ/ton? But I'm not sure the other three. Probably have to go to the aglu data documentation.